src/components/Layout/Layout.tsx

```
@@ -1,50 +1,24 @@
 
-const variants = {
-    hidden: { opacity: 0, x: -150, y: 0 },
-    enter: { opacity: 1, x: 0, y: 0 },
-    exit: { opacity: 0, x: 0, y: -100 },
-};
 
-const SiteWrapper = styled.div`
-    min-height: 100vh;
-`;
 
 const Layout: React.FC<ReactProps> = ({ children }) => {
```

ESLint

[error] More than 1 blank line not allowed.

(style/no-multiple-empty-lines)